### PR TITLE
Update to SK 1 beta 7

### PR DESCRIPTION
**Pull request by glahaye**
### Motivation and Context
Getting the latest and greatest

### Description
Update nugets

### Contribution Checklist
- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Read more about plugin authentication here: [Plugin authentication](https://plat
 ### Prerequisites
 
 1. The name of your plugin. This should be identical to the `NameForHuman` in your plugin manifest.
-   > Please refer to OpenAI for the [manifest requirements](https://platform.openai.com/docs/plugins/getting-started/plugin-manifest).
```
Comment (TaoChenOSU) [inline plugins/README.md:31]: What is wrong with this original link?

File: plugins/shared/PluginManifest.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace Plugins.PluginShared;
 
 /// <summary>
 /// This class represents the OpenAI plugin manifest:
-/// https://platform.openai.com/docs/plugins/getting-started/plugin-manifest
+/// https://platform.openai.com/docs/plugins/getting-started
```
Comment (TaoChenOSU) [inline plugins/shared/PluginManifest.cs:9]: Same as above.

File: plugins/README.md (hunk 1 above)
Comment (glahaye) [reply plugins/README.md:31]: The old link now redirects to the new one.

File: plugins/shared/PluginManifest.cs (hunk 2 above)
Comment (glahaye) [reply plugins/shared/PluginManifest.cs:9]: The old link now redirects to the new one.

File: plugins/README.md (hunk 1 above)
Comment (TaoChenOSU) [reply plugins/README.md:31]: The old one goes to a session under that page

**Review by momuno — APPROVED** [review]

**Pull request merged**